js/runtime.js

```diff
@@ -1192,7 +1212,25 @@
                 var date = new Date(prevDate.valueOf());
                 date.setFullYear(date.getFullYear() + years);
                 return date;
-            }
+            },
```

Comment:
test/all-blocks-have-function.js reported these functions missing so I put them in. Probably should have made a different PR for that.